src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,53 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static addToAlive(animalObj) {
+    this.alive.push(animalObj);
+  }
+
+  static removeFromAlive(animalObj) {
```

Comment:
- let's use more descriptive names for your variables. It should be maximally clear what your method does by its name
```suggestion
  static removeDeadAnimal(animalObj) {
```

Reply:
also, you should not use data type in the name of your variables, as it does not give us any additional information. It should have clear naming even without explicit mention of data type

Reply:
Thank you for suggestion!